.github/workflows/main.yml

```diff
@@ -27,3 +28,19 @@ jobs:
       - name: Regal Lint
         if: ${{ !cancelled() }}
         run: regal lint --format github .
+
+  deploy:
+    name: Deploy policies
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    needs: lint
+    if: success() && github.ref == 'refs/heads/main'
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Upload
+        env:
+          PHYLUM_TOKEN: ${{ secrets.PHYLUM_POLICY_TOKEN }}
```

Comment:
It might help us in the future if there is a comment here specifying which account this token comes from.

Reply:
I could reference the details, but I think those details will be moved within a couple weeks.